lib/rubocop/cop/naming/rescued_exceptions_variable_name.rb

```diff
@@ -150,6 +153,13 @@ def message(node)
           preferred_name = preferred_name(offending_name)
           format(MSG, preferred: preferred_name, bad: offending_name)
         end
+
+        def check_shadowed_variable_name(node)
+          node.each_descendant do |n|
```

Comment:
It seems that descendant nodes can be limited.

```suggestion
          node.each_descendant(:lvar) do |n|
```